src/Containers/MinimalContainers/RecordArray.hpp

```diff
@@ -0,0 +1,94 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2020 QMCPACK developers.
+//
+// File developed by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//
+// File created by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef QMCPLUSPLUS_RECORDARRAY_H
+#define QMCPLUSPLUS_RECORDARRAY_H
+
+#include <stdio.h>
```

Comment:
Is this really needed? If yes, ue C++ header <cstdio> is better.

Reply:
removed